### DISCUSSION TITLE
2011/04/23/%E6%88%91%E7%9A%84%E8%AF%81%E4%BB%B6%E7%85%A7

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#327] [Announcements]
# 2011/04/23/%E6%88%91%E7%9A%84%E8%AF%81%E4%BB%B6%E7%85%A7

今天翻看我的旧照片时，发现我还保留着几张小时候的证件照。于是把他们翻拍出来，依次排好，也是我成长轨迹的一个见证。

https://blog.qizhen.xyz/2011/04/23/%E6%88%91%E7%9A%84%E8%AF%81%E4%BB%B6%E7%85%A7

**Comment by ruanqizhen** [comment]
之前博客下的留言：


长生天
2011年05月1日 19:49 编辑
你的相片说明你年纪不小了。怎么孩子还这么小那！

回复
ruanqizhen
2011年05月3日 08:59 编辑
俺们大器晚成呗 :)

回复
Jiucheng Nie
2011年04月30日 04:10 编辑
您好，我是一名硕士在读生，请问，如果我想到NI公司从事LabView工程师的工作，应该具备哪些方面的知识呢？例如我需要掌握哪些编程语言或哪些软件的操作？请阮工指点一下，我好朝着这个方向努力

回复
ruanqizhen
2011年05月3日 08:55 编辑
可能你用到的每个面试官或者应聘的不同职位都有不同的要求。我觉得要给面试官留下好印象主要还不在于学过哪一门编程语言，而在于有没有把曾经学过的东西学精通。如果能把现在正在研究的课题搞得很透彻，面试就不怕了。

回复
Jiucheng Nie
2011年05月6日 23:51 编辑
谢谢您，当初本科毕业设计做的课题就是基于LabView的一套血压采集处理系统，然后读了您写的书，于是就对LabView的工作特别向往，非常感谢您的回复，我打算再把LabView学的更透彻一些，另外再把C和C++捡起来

回复
Jiucheng Nie
2011年04月30日 04:04 编辑
您好

回复
rabbitlee1982
2011年04月29日 22:02 编辑
还是幼儿园的最好看~

回复
淘宝女装推荐
2011年04月26日 16:48 编辑
哈哈，表情很淡定阿

回复
勺子
2011年04月24日 21:25 编辑
你LP說得很對挖。。。

回复
haiyingjia
2011年04月24日 19:35 编辑
你高中时长的比这张好多呢,顶多是盗窃犯吧

回复
